index.html

```diff
@@ -30,6 +40,15 @@
       "@context": "https://schema.org"
     }
   </script>
+  <!-- Global site tag (gtag.js) - Google Analytics -->
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-XT0RMBXMZM"></script>
+  <script>
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+
+    gtag('config', 'G-XT0RMBXMZM');
+  </script>
```

Comment:
This appears twice. We are instancing it in head.html include too. I don't see where we are using `include head` so maybe it doesn't appear duplicated in the site. I would **make user we are using only one of inline head in index.html or including the head.html but not maintaining both files**